view/frontend/templates/breadcheckout/price.phtml

```diff
@@ -0,0 +1,150 @@
+<?php /** @var $block Bread\BreadCheckout\Block\Product\View */ ?>
+<?= /* @noEscape */ $block->getBeforeHtml(); ?>
+
+<div class="splitpay-clickable-price" style="padding-bottom: 5px"></div>
+
+<script type="text/javascript">
+    require(['jquery', 'Bread_BreadCheckout/js/product/splitpay'], function($, splitpay) {
+
+        document.splitpayconfigureButton = function (items) {
+            var buttonCss           = '<?= /* @noEscape */ $block->getButtonDesign(); ?>';
+            var financingInfo   = <?= /* @noEscape */ $block->getFinancingJson(); ?>;
+            var breadConfig = {
+                <?php if (!$block->isHealthcare()) : ?>
+                items: items,
+                <?php else : ?>
+                customTotal: items[0]['price'],
+                <?php endif; ?>
+                discounts: <?= /* @noEscape */ $block->getDiscountDataJson(); ?>,
+                allowCheckout: false,
+                done: function() {},
+
+                /**
+                 * Add to cart call back used when allow checkout from product page = false
+                 *
+                 * @param callback
+                 */
+                addToCart:  function(callback) {
+                    document.getElementById('product_addtocart_form').submit();
+                }
+            };
+
+            if (buttonCss !== null && buttonCss !== '' && buttonCss != false) {
```

Comment:
would it be enough to check if `buttonCss` is truthy here or is there some edge case?